src/permission/tests/test_utils/__init__.py

```diff
@@ -1,4 +1,8 @@
-from test_logics import *
-from test_handlers import *
-from test_permissions import *
-from test_field_lookup import *
+import django
+if django.VERSION < (1, 6):
+    # Django 1.5 and earlier use a different test method and the followings
+    # are required to be exposed
+    from permission.tests.test_utils.test_logics import *
+    from permission.tests.test_utils.test_handlers import *
+    from permission.tests.test_utils.test_permissions import *
+    from permission.tests.test_utils.test_field_lookup import *
```

Comment:
In Django 1.9, test runner automatically detect test cases so you don't need to import explicitly.